src/components/DownloadLoginFile.css

```diff
@@ -22,7 +22,7 @@
 }
 
 .download-loginfile .long-touch-indicator.animate {
-    animation: long-touch-indicator-animation .7s linear forwards;
+    animation: long-touch-indicator-animation 800ms linear forwards;
```

Comment:
So you actually changed it to 800 :D

Reply:
Had to change it back to 700ms because the indicator did not fill up completely with 800. I also added a comment explaining this for future generations.